headers/functions/overlay29.h

```diff
@@ -72,6 +83,7 @@ int GetMovePower(struct entity* entity, struct move* move);
 void AddExpSpecial(struct entity* attacker, struct entity* defender, int base_exp);
 void EnemyEvolution(struct entity* enemy);
 void EvolveMonster(struct entity* monster, undefined4* param_2, enum monster_id new_monster_id);
+bool DisplayActions(undefined param_1);
```

Comment:
The parameter gets written to `dungeon::current_active_entity` under certain conditions, so I'm pretty sure we can define it as a `struct entity*`

Reply:
You're right, I'll update it.